src/ModelInit.jl

```diff
@@ -671,7 +671,7 @@ function trim_source_tiles(
       has_source = s in tile_sources
       bright_pixels = Bool[];
       if has_source
-        #println("Tiles $h $w has source $s")
+        info("Tiles $h $w has source $s")
```

Comment:
I don't think you want to display this -- it was just for debugging.  My bad for commenting it out and not just deleting it.